scripts/create_native_image_test_env.sh

```diff
@@ -0,0 +1,215 @@
+#!/bin/bash
+
```

Comment:
Can you add source code comment that explains the purpose of this script?

Reply:
Good idea, done. I was also thinking of adding a few instructions on usage. Do you think this would be a good location? https://github.com/googleapis/gapic-generator-java/blob/main/DEVELOPMENT.md